src/layouts/Conversation/index.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import Head from "next/head";
+import Link from "next/link";
+import MessageIcon from '@mui/icons-material/Message';
+import { makeStyles } from "@mui/styles";
+
+const useStyles = makeStyles({
+ // >>>>>>>>>>>>>>>  Interested  >>>>>>>>>>>>>>>>>>>>>>>>>>
+
+ interest:{
+  backgroundColor:"white",
+  textAlign: "center",
+  margin:"150px 10px",
+  textAlign: "-webkit-center",
+  
+  '& p':{
+      fontSize: "20px",
+      margin:"40px",
+  },
+  '& .MuiSvgIcon-root': {
+      width: '60px',
+      textAlign: "center",
+      alignItems:"center",
+      height: "25px",
+      margin:"-9px",
+  }
+},
+start_btn: {
+  color: "#6e07f3",
+  border:"2px solid #6e07f3",
+  borderRadius:"35px",
+  maxWidth:"250px",
+  height:"40px",
+  padding:"10px",
+  "&:hover": {
+  backgroundColor: "#6e07f3",
+  color:"white",
+},},
+});
+
+function Conversation() {
+  const classes = useStyles();
+  return (
+    <div style={{backgroundColor:"white"}}>
+      <Head>
+        <title>Home-Hashtag404</title>
+        <meta
+          name="description"
+          content="Meta description for the Home page THIS IS AS HOME PAGE" />
+        <meta name="keywords" content="test1, test2, ssr, nextjs" />
+      </Head>
+ {/* >>>>>>>>>>>>>>>     Interested       >>>>>>>>>>>>>>>>>>>>>>>>>>>> */}
```

Comment:
please use /*your comment */

Reply:
it did't work like thar
i used /*...*/ this but it did't work thats why i used {/*...*/} this